var/spack/repos/builtin/packages/libtheora/package.py

```diff
@@ -40,7 +67,64 @@ def autoreconf(self, spec, prefix):
         else:
             sh("./autogen.sh", "prefix={0}".format(prefix))
 
-    def configure_args(self):
-        args = []
-        args += self.enable_or_disable("doc")
-        return args
+
+class MSBuildBuilder(MSBuildBuilder):
+    def is_64bit(self):
+        return platform.machine().endswith("64")
+
+    def setup_build_environment(self, env):
+        spec = self.pkg.spec
+        env.set("SPACK_OGG_PREFIX", spec["libogg"].prefix)
+        # devenv is needed to convert ancient MSbuild project to modern
+        # msbuild project so MSBuild versions older than 2010 can build this
+        # project
+        devenv_path = os.path.join(self.pkg.compiler.vs_root, "Common7", "IDE")
+        env.prepend_path("PATH", devenv_path)
+
+    @property
+    def build_directory(self):
+        win_dir = os.path.join(super().build_directory, "win32")
+        vs_dir = "VS2008"
+        return os.path.join(win_dir, vs_dir)
+
+    @property
+    def toolchain_version(self):
```

Comment:
I have a similar question here as for https://github.com/spack/spack/pull/35095#discussion_r1096461931 - I don't see this method being used anywhere in this PR.

Reply:
Response is the same as it is for #35095, this method is required by the builder.